Tools/CISupport/ews-build/steps.py

```diff
@@ -2255,6 +2271,310 @@ def hideStepIf(self, results, step):
         return not self.doStepIf(step)
 
 
+class RemoveAndAddLabels(buildstep.BuildStep, GitHubMixin, AddToLogMixin):
+    name = 'remove-and-add-labels'
+    flunkOnFailure = False
+    haltOnFailure = False
+
+    def __init__(self, label_to_add='', labels_to_remove=[GitHubMixin.SAFE_MERGE_QUEUE_LABEL], **kwargs):
```

Comment:
You don't want to store a list in a default argument like this, it does weird things if you start modifying the list (since Python prefers to pass lists by reference instead of by value). Instead, you should do this:
```
def __init__(self, label_to_add='', labels_to_remove=None, **kwargs):
    self.label_to_add = label_to_add
    self.labels_to_remove = [GitHubMixin.SAFE_MERGE_QUEUE_LABEL] if labels_to_remove is None else labels_to_remove
    super().__init__(**kwargs)
```